### PR TITLE
Ashwalker tendrils now drop everything on victims before consuming

### LINKED ISSUE #30071: Ashwalker necropolis destroys objects gibbing when they shouldn't
The ashwalker necropolis no longer drops _***all**_ the contents of its victims, objects namely referring to belt slot objects & bags and chest gibs are destroyed wheras they weren't before. Anything worn on the torso is generally destroyed.

This is a deviant behaviour because ashwalker tendrils are supposed to not destroy the contents of its sacrifices after it was summarily changed to preserve its contents. Suggesting that the code for gibbing is obsolete or needs tweaking.

### PR DESCRIPTION
**Pull request by silicons**
Fixes #30071

### CODE REVIEW

**Comment by XDTM** [comment]
use the drop_everything() proc, i made it for these cases

File: code/modules/ruins/objects_and_mobs/ash_walker_den.dm [hunk 1]
```diff
@@ -25,10 +25,14 @@
...
 			playsound(get_turf(src),'sound/magic/demon_consume.ogg', 100, 1)
+			for(var/obj/item/W in H)
+				if(!H.dropItemToGround(W))
+					qdel(W)
 			if(ismegafauna(H))
 				meat_counter += 20
 			else
 				meat_counter++
+			H.regenerate_icons()	//Just incase it fails to gib
 			H.gib()
```
Comment (ChangelingRain) [inline code/modules/ruins/objects_and_mobs/ash_walker_den.dm]: doesn't gibbing already do this, though?

**Comment by kevinz000** [comment]
@XDTM i dont see it

**Comment by XDTM** [comment]
I swear i added it, but i can't find it either...

Can you make it an inventory proc anyway? There are other situations where this'd be nice

**Comment by Bawhoppen** [comment]
This is dumb and another powergame-friendly feature :-1:

**Pull request merged**